test/DynamicDependency/data/Microsoft.WindowsAppSDK.Framework/appxmanifest.xml

```diff
@@ -53,16 +53,22 @@
         <ActivatableClass ActivatableClassId="Microsoft.Windows.AppLifecycle.AppLifecycle" ThreadingModel="both" />
       </InProcessServer>
     </Extension>
-    <Extension Category="windows.activatableClass.inProcessServer">
-      <InProcessServer>
-        <Path>Microsoft.WindowsAppSDK.dll</Path>
-        <ActivatableClass ActivatableClassId="Microsoft.Windows.PushNotifications.PushNotificationChannel" ThreadingModel="both" />
-        <ActivatableClass ActivatableClassId="Microsoft.Windows.PushNotifications.PushNotificationCreateChannelResult" ThreadingModel="both" />
-        <ActivatableClass ActivatableClassId="Microsoft.Windows.PushNotifications.PushNotificationActivationInfo" ThreadingModel="both" />
-        <ActivatableClass ActivatableClassId="Microsoft.Windows.PushNotifications.PushNotificationReceivedEventArgs" ThreadingModel="both" />
-        <ActivatableClass ActivatableClassId="Microsoft.Windows.PushNotifications.PushNotificationRegistrationToken" ThreadingModel="both" />
-        <ActivatableClass ActivatableClassId="Microsoft.Windows.PushNotifications.PushNotificationManager" ThreadingModel="both" />
-      </InProcessServer>
-    </Extension>
+      <Extension Category="windows.activatableClass.inProcessServer">
+          <InProcessServer>
+              <Path>Microsoft.WindowsAppSDK.dll</Path>
+              <ActivatableClass ActivatableClassId="Microsoft.Windows.PushNotifications.PushNotificationChannel" ThreadingModel="both" />
+              <ActivatableClass ActivatableClassId="Microsoft.Windows.PushNotifications.PushNotificationCreateChannelResult" ThreadingModel="both" />
+              <ActivatableClass ActivatableClassId="Microsoft.Windows.PushNotifications.PushNotificationActivationInfo" ThreadingModel="both" />
+              <ActivatableClass ActivatableClassId="Microsoft.Windows.PushNotifications.PushNotificationReceivedEventArgs" ThreadingModel="both" />
+              <ActivatableClass ActivatableClassId="Microsoft.Windows.PushNotifications.PushNotificationRegistrationToken" ThreadingModel="both" />
+              <ActivatableClass ActivatableClassId="Microsoft.Windows.PushNotifications.PushNotificationManager" ThreadingModel="both" />
+          </InProcessServer>
+      </Extension>
+      <Extension Category="windows.activatableClass.inProcessServer">
+          <InProcessServer>
+              <Path>Microsoft.WindowsAppSDK.dll</Path>
+              <ActivatableClass ActivatableClassId="Microsoft.Windows.System.Power.PowerManager" ThreadingModel="both" />
+          </InProcessServer>
```

Comment:
You'll also need this in build\NuSpecs\AppxManifest.xml

That's what goes into the Transport package for use by the aggregator when making Microsoft.WindowsAppSDK.1.0*.msix.

The file here's for a test-equivalent WASfwk package for use by tests in this repo.

P.S. You'll notice the `<InProcessServer>` declarations differ slightly, 1 `<Extension>` in the Transport package's manifest for all servers in the dll, but 1 `<Extension>` here for each functional group for our tests. This is intentional.

Reply:
Just curios: why the disparity?

Reply:
The test package has multiple `<InProcessServer>` specifically to ensure DynDep's tests exercise that (we had a bug a while back it didn't handle >1 correctly).

But it's slightly less efficient, so for Transport (which is a fragment feeding into product) it's slightly more efficient to have the 1 extension.